lib/cosmos/tools/cmd_sender/cmd_sender.rb

```diff
@@ -619,13 +637,17 @@ def update_cmd_params(ignored_toggle = nil)
               end
             end
             state_value_item.setTextAlignment(Qt::AlignRight | Qt::AlignVCenter)
-            state_value_item.setFlags(Qt::NoItemFlags | Qt::ItemIsSelectable | Qt::ItemIsEnabled | Qt::ItemIsEditable)
+            if @production
+              state_value_item.setFlags(Qt::NoItemFlags)
+            else
+              state_value_item.setFlags(Qt::NoItemFlags | Qt::ItemIsSelectable | Qt::ItemIsEnabled | Qt::ItemIsEditable)
+            end
             @@table.setItem(row, 2, state_value_item)
 
-            # If the parameter is required set the combobox to MANUAL and
+            # If the parameter is required clear the combobox and
             # clear the value field so they have to choose something
             if packet_item.required && !old_params[packet_item.name]
-              value_item.setText(MANUALLY)
+              value_item.setText('')
```

Comment:
This is the only subtle change that affects the non-production version of CmdSender. Simply display a blank combo instead of MANUALLY ENTERED which I think is an improvement because it shows visually that you have to select something.